sys/include/net/ng_sixlowpan.h

```diff
@@ -10,6 +10,56 @@
  * @defgroup    net_ng_sixlowpan    6LoWPAN
  * @ingroup     net
  * @brief       6LoWPAN implementation
+ *
+ * This module is for usage with the @ref net_ng_netapi.
+ *
+ * # @ref net_ng_netapi command documentation
+ *
+ * ## `NG_NETAPI_MSG_TYPE_RCV`
+ *
+ * @ref NG_NETAPI_MSG_TYPE_RCV expects a packet in receive order
+ * (payload first, headers last) with the payload marked with
+ * @ref NG_NETTYPE_SIXLOWPAN. Every other packet will be silently discarded.
+ *
+ * If the first byte of the payload is @ref NG_SIXLOWPAN_UNCOMPRESSED this byte
+ * will be removed, the remaining payload will be marked with
```

Comment:
Will the payload really shortened by one byte, or is this first byte overwritten?